src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -50,11 +50,13 @@
 import jakarta.ws.rs.core.Request;
 import jakarta.ws.rs.core.Response;
 import jakarta.ws.rs.core.Response.Status;
+import java.util.ArrayList;
```

Comment:
These two new imports are actually unused. This is strange, I would have expected CheckStyle to complain about it. Perhaps there is no rule against unused imports 🤔